app/extras/poll_exporter.rb

```diff
@@ -0,0 +1,118 @@
+class PollExporter
+
+  include Routing
+
+  def initialize (poll)
+    @poll = poll
+  end
+
+  def file_name
+    "#{@poll.title.parameterize}-export.csv"
+  end
+
+  def poll_info (current_user)
+     PollEmailInfo.new(recipient: current_user, event: @poll.created_event, action_name: :export)
+  end
+
+  def label (key, *opts)
+    I18n.t("poll.export.#{key.downcase}".to_sym, *opts)
+  end
+
+  def meta_table
+    outcome = @poll.current_outcome
+    engagement = nil
+    voted = @poll.stances_count
+    total = @poll.members.count
+    engagement =  label('percent_voted', num:voted ,denom:total, percent:"#{(voted*100/total)}%")
+
+    {
+      title: @poll.title,
+      author: @poll.author.name,
+      created_at: @poll.created_at,
+      closing_at:  (@poll.closing_at unless @poll.closed_at),
+      closed_at: @poll.closed_at,
+      engagement:engagement,
+      stances: @poll.stances_count,
+      participants: @poll.members.count,
+      details: @poll.details,
+      group_name: @poll.group.full_name,
+      outcome_author: outcome&.author,
+      outcome_created_at: outcome&.created_at,
+      outcome_statement: outcome&.statement,
+      poll_url: poll_url(@poll)
+    }.compact
+  end
+
+  def stance_matrix
+    rows = []
+    rows << [label("Participant"), @poll.poll_options.map(&:display_name), label("Reason")].flatten
```

Comment:
Yep so I think this would be
```
label(:participant)
```
which would go to
```
I18n.t("poll.export.participant")
```
which would get translated to "Participant" (or "Partícipe", etc.)